product/tests/test_services/test_item_creator.py

```
@@ -0,0 +1,159 @@
+from django.core.exceptions import ObjectDoesNotExist
+from django.http import Http404
```

Ruff (0.8.2)

`django.http.Http404` imported but unused

Remove unused import: `django.http.Http404`

(F401)